exonum/src/messages/mod.rs

```diff
@@ -194,7 +251,7 @@ impl Message {
     }
 
     /// Checks buffer and return instance of `Message`.
```

Comment:
Checks... and returns...
or
Check... and return...

Reply:
Fixed